google/cloud/security/inventory/inventory_loader.py

```diff
@@ -274,6 +286,13 @@ def _send_email(organization_id, cycle_time, cycle_timestamp, status, pipelines,
     except util_errors.EmailSendError:
         LOGGER.error('Unable to send email that inventory snapshot completed.')
 
+
+def _configure_logging(configs):
+    """Configures the loglevel for all loggers."""
+    desc = configs.get('loglevel')
+    level = LOGLEVELS[desc]
```

Comment:
Hello, I can't push code to this PR but hopefully you can avoid a `KeyError`  here by using a default fallback value:

```
level = LOGLEVELS.setdefault(desc, 'info')
```